test/test_functions

```diff
@@ -1383,17 +1383,18 @@ set_valgrind_options() {
 # @param workspace  absolute path to the directory where the benchmark will put the files in
 setup_benchmark_environment() {
   local workspace="$1"
-  mkdir -p /cvmfs/$FQRN
+  local me=$(whoami)
+  sudo sh -c "mkdir -p /cvmfs/$FQRN && chown $me /cvmfs/$FQRN"
   mkdir -p "$CVMFS_OPT_OUTPUT_DIR/$FQRN"
   mkdir -p $workspace/cache
-
-  # cleaning the cache
-  clean_benchmark_cache
-
+  
+  # make sure the user can execute /bin/fusermount, write into /dev/fuse and write into /cvmfs/<repo>
+  sudo sh -c "usermod -a -G fuse $me"
```

Comment:
There is a problem with this: if the user was not in the "fuse" group then it's not effectively added in the current session, so the execution will fail the very first time because it won't be able to use /bin/fusermount. Any clues about how to solve this?